sample-app/src/components/InputDropdown.tsx

```diff
@@ -80,9 +87,36 @@ export default function InputDropdown({
     setScreenReaderKey(0);
   }
 
+  const screenReaderPhrases: string[] = [];
+  if (options.length < 1) {
+    const phrase = processTranslation({
+      phrase: `0 autocomplete option found.`,
+      pluralForm: `0 autocomplete options found.`,
+      count: 0
+    })
+    screenReaderPhrases.push(phrase);
+  } else {
+    options.forEach(section => {
+      const phrase = section.label ?
+        processTranslation({
+          phrase: `${section.results.length} ${section.label} autocomplete option found.`,
```

Comment:
can we group these two process translation into one? like either construct `${section.results.length} ${section.label}` outside then call processTranslation or check the label directly in string and add '' if it doesn't exist?

Reply:
I would imagine that `processTranslation` will eventually work like the same function we have in SDK use for  translations, which includes interpolation and pluralization. Maybe we can keep the call for no results

Reply:
Makes sense. I added it back for no results